language_detector/main.py

```diff
@@ -5,5 +5,21 @@
 
 def detect_language(text, languages=LANGUAGES):
     """Returns the detected language of given text."""
-    # implement your solution here
-    pass
+    counter = {}
+    for diction in languages:
+        counter[diction['name']] = 0
+        for word in diction['common_words']:
+            for text_word in text.split():
+                if word == text_word:
+                    counter[diction['name']] += 1
+    
+    return max(counter, key = counter.get)
```

Comment:
Great idea to use max! This is much better than the complicated sort we ended up using.